### PR TITLE
OSPP

### PR DESCRIPTION
**Pull request by Gui-Yue**
* add build script for docker building.
* Complete multi-container merging(Add riscv support)

### CODE REVIEW

File: .gitmodules
Comment (jack30724) [inline .gitmodules:1]: @Gui-Yue   It’s better to change the following URL to the official repository address.

Comment (jack30724) [inline .gitmodules:1]: @Gui-Yue   It’s better to change the following URL to the official repository address:

url = https://github.com/Gui-Yue/space-AllToOne.git

**Comment by jack30724** [comment]
Related PRs for this OSPP project:

https://github.com/ao-space/ao.space/pull/11
https://github.com/ao-space/space-single/pull/2
https://github.com/ao-space/space-agent/pull/29

**Review by jack30724 — APPROVED** [review]

**Comment by jack30724** [comment]
@jianminww     

This PR is part of the "Merge Multiple Containers into a Single Container" project (https://summer-ospp.ac.cn/org/prodetail/24c570216?lang=zh&list=pro) for the Summer-ospp. 

I have reviewed it. 

It's ready for merging, please proceed with the merge.

Related PRs for this OSPP project:
https://github.com/ao-space/ao.space/pull/11
https://github.com/ao-space/space-single/pull/2
https://github.com/ao-space/space-agent/pull/29

**Comment by jimmieme** [comment]
@Gui-Yue @jack30724 Thank you for your contribution.

**Pull request merged**